lib/persistize/active_record.rb

```diff
@@ -76,7 +76,7 @@ def #{callback_name}                                                     # def _
             parent = #{self.name}.find(parent_id)                                  #   parent = Project.find(parent_id)
             parent.#{update_method}!                                               #   parent._update_completed!
           end                                                                      # end
-          after_save :#{callback_name}                                             # after_save :_update_completed_in_project_callback
+          after_commit :#{callback_name}, on: [:create, :update]                   # after_commit :_update_completed_in_project_callback, on: [:create, :update]
```

Comment:
Looks good to me but shouldn't we be moving the `after_destroy` callback to `after_commit` too? Same per the below changes

Reply:
I don't know if we can, `after_commit` the record is deleted and there is no info in database about it, but maybe we can give it a try.

Reply:
👌 sounds sensible, let's leave it as it is currently